### PR TITLE
Remove use of 'resetData' in ITs

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

**Comment by julienlancelot** [comment]
Do not forget to upgrade orchestrator to 3.31.0.2646

File: its/src/test/java/com/sonar/it/jenkins/SonarPluginTest.java [hunk 1]
```diff
@@ -122,7 +124,11 @@ public void setUp() {
 
   @After
   public void cleanup() {
+    if (!wsClient.projects().search(new SearchRequest().setProjects(Collections.singletonList(MVN_PROJECT_KEY))).getComponentsList().isEmpty()) {
```
Comment (julienlancelot) [inline its/src/test/java/com/sonar/it/jenkins/SonarPluginTest.java:127]: There's another interesting way to replace the resetDat() => it's to do a bulk delete of all projects.
See how the languages team has do it for instance here : https://github.com/SonarSource/sonar-dotnet/blob/98c0671a0b71ec36f1fc7a237aa51a7b6e5f57d7/its/src/test/java/com/sonar/it/shared/TestUtils.java#L187

**Review by julienlancelot — APPROVED** [review]
Please upgrade orchestrator, and feel free to apply my comment on the way to purge all projects at once.

File: its/src/test/java/com/sonar/it/jenkins/SonarPluginTest.java (hunk 1 above)
Comment (dbmeneses) [reply its/src/test/java/com/sonar/it/jenkins/SonarPluginTest.java:127]: nice, I'll use that

**Pull request merged**